src/main/scala/cromwell/engine/backend/jes/OperationMetadata.scala

```diff
@@ -4,27 +4,41 @@ import com.typesafe.scalalogging.LazyLogging
 import com.google.api.services.genomics.model.Operation
 import org.joda.time.DateTime
 import scala.collection.JavaConverters._
+import scala.util.{Failure, Success, Try}
 
 object OperationMetadata extends LazyLogging {
+
+  private def dateStringToDateTime(dateTime: Option[AnyRef]): Try[DateTime] = {
+    val tryValue: Try[String] = dateTime map { _.asInstanceOf[String] } match {
+      case Some(x) => Success(x)
+      case None => Failure(new NoSuchFieldException())
```

Comment:
I think the thing here is that we want to be robust to situations where they're not actually supplying us a date when we expect it - not that we should throw an exception